plugins/test/unit/plugins/distributors/yum/metadata/test_updateinfo.py

```diff
@@ -1,14 +1,12 @@
 """
 Tests for the pulp_rpm.plugins.distributors.yum.metadata.updateinfo module.
 """
+import copy
+import mock
```

Comment:
Since this isn't in the standard library (at least as of python2), it should go in the middle group of imports per pep8.